src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -231,6 +231,42 @@ bool MemoryViewerControlBinding::HandleNavigation(UINT nChar)
             }
             return true;
 
+        case VK_ADD:
+        case VK_SUBTRACT:
+        {
+            auto nAddress = m_pViewModel.GetAddress();
+            const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+            auto iMem = pEmulatorContext.ReadMemory(m_pViewModel.GetAddress(), m_pViewModel.GetSize());
+            auto iMaxValue = (1ULL << (ra::data::MemSizeBytes(m_pViewModel.GetSize()) * 8)) - 1;
+            auto iModifier = 0x1;
+
+            if (bControlHeld)
+                iModifier *= 4;
+            if (bShiftHeld)
+                iModifier *= 8;
```

Comment:
It might make more sense to have shift increment/decrement by 16 (update the high nibble).